lms/lib/courseware_search/test/test_lms_filter_generator.py

```diff
@@ -266,6 +358,44 @@ def test_content_group_with_cohort_not_provided(self):
         self.assertEqual(unicode(self.courses[0].id), field_dictionary['course'])
         self.assertEqual(None, filter_dictionary['content_groups'])
 
+    def test_split_test_with_content_groups_user_not_assigned(self):
```

Comment:
I'd create a new test class for all the partition tests, as that would allow the individual tests to have clearer names.